Demo/imdb/queries.py

```diff
@@ -0,0 +1,48 @@
+actors_played_with_nicolas_cage_query = \
```

Comment:
Each query, should be an object, with the following attributes:
1. Actual query
2. Description
3. Expected result
4. Expected run-time

Reply:
ok